amr-wind/wind_energy/ABLFieldInit.cpp

```diff
@@ -126,6 +141,97 @@ void ABLFieldInit::operator()(
             velocity(i, j, k, 1) += vfac * damp * z * std::cos(bval * xl);
         }
     });
+
+#ifdef AMR_WIND_USE_NETCDF
+    // Skip fine levels and interpolate data from already loaded coarse levels
```

Comment:
can you move this into it's own function kind of like perturb_temperature or init_tke. You could also move this into its own physics class but then you'll have to be careful about order in the input file and call it after ABL... maybe the physics class could check if ABL was created first otherwise abort?

Reply:
another option is to make another field initializer kind of like field_init. Then this can go in it's own class and you won't have to worry about ordering physics classes